packages/amplify_analytics_web/lib/interop/amplify_analytics_js_interop.dart

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+@JS('aws_amplify')
+library amplifycore;
+
+import 'package:js/js.dart';
+// import 'package:amplify_auth_cognito_web/interop/types/SignUp/CognitoSignUpParamImpl.dart';
+// import 'package:amplify_auth_cognito_web/interop/types/SignIn/CognitoSignInParamImpl.dart';
+
+@JS('Analytics')
+
+/// Interop class for the amplify-js [Analytics category](https://github.com/aws-amplify/amplify-js/tree/main/packages/analytics)
+abstract class AnalyticsJS {
+  /// Interop member for Analytics.signIn
+  external static AnalyticsJS record(attributes);
+
+  /// Interop member for Analytics.enable
+  external static AnalyticsJS enable();
+
+  /// Interop member for Analytics.disable
+  external static AnalyticsJS disable();
+
+  // example from auth
```

Comment:
Again, probably not that important just for this POC and getting the code together, but this could be removed.